Luciefifi/my-brand-backend-api
# chore(database):

# WHAT DOES THIS PR DO
Setting up MongoDB database

#DESCRIPTION OF THE TASK TO BE COMPLETED
* Go to mongodb.com
* create a database
* create username and password 
* install dependencies (mongoose )
* create .env , .env.example and .gitignore files
* Put your MongoDB url in .env file
* move .env into gitignore
* create dBase.js file to create database connection
* connect to database
* Test connection
* [Finishes #001]

# How should this manually be tested 
when you are done with cloning the repository into your machine ad checked that you are in the right folder , go ahead and open your terminal and type **npm install** to install all dependencies and node_modules  , when all of that are done still in your terminal type **npm run dev** to open the server and test whether database has been connected successfully  , if it did in your terminal you will see the message "database connected"

# Relevant pivotal tracker story
story id :  #001
